components/shopify/actions/create-order/create-order.mjs

```diff
@@ -0,0 +1,123 @@
+import shopify from "../../shopify.app.js";
+
+export default {
+  key: "shopify-create-order",
+  name: "Create Order",
+  description: `Creates a new order
+    More information at [Shopify Order Object](https://shopify.dev/api/admin-rest/2022-01/resources/order#resource_object)`,
+  version: "0.0.1",
+  type: "action",
+  props: {
+    shopify,
+    lineItems: {
```

Comment:
**Comment**
I attempted to rework the `lineItems` prop using the `additionalProps` component property, as mentioned in [this slack thread](https://pipedream-users.slack.com/archives/G01J50YGJGP/p1642196062002400?thread_ts=1632172750.006200&cid=G01J50YGJGP). But I had an issue with prop values being reset when a previous prop with async options was changed. I think the current approach works better for now.

![8258805a6e](https://user-images.githubusercontent.com/19861096/151212283-9cc9f8c5-a2b2-4132-9011-64553e3d34ae.gif)